### DISCUSSION TITLE
How to get the tab title?

### DISCUSSION THREAD

**Discussion by awlobo** [michaelbrusegard/tabline.wez#29] [Q&A] [answered]
Hello! Is there a way to display the tab name? I can see it is not included in https://github.com/michaelbrusegard/tabline.wez#available-components but I'd like to know if it's possible to retrieve it through a function or any other way.
Thanks! Cool work and super useful plugin 💪🏻

**Comment by michaelbrusegard** [comment, marked as answer]
There is currently not a component that supplies the tab title. It should be a pretty simple change to add. You can create an issue with a feature request if you want

**Reply by awlobo** [reply]
Will do, thanks!
